### PR TITLE
Ensure all CRDs for cert-manager are installed

### PR DESCRIPTION
**Pull request by nikkelma**
Terraform provider `banzaicloud/k8s` only parses the first non-empty YAML document when multiple documents exist in the same file (split by `---`). Applied workaround of creating multiple manifest resources; tested with cert-manager 0.15.1 CRDs YAML and found manifest contents containing no YAML document (empty or only comments) caused an error. Second workaround applied of downloading and trimming CRDs YAML to make Terraform logic simple.

Linked issue #112

### CODE REVIEW

File: rancher-common/README.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ Kubernetes version to use for Rancher server RKE cluster
 - Default: **`"0.12.0"`**
 Version of cert-mananger to install alongside Rancher (format: `0.0.0`)
 
-Used in the case of older Rancher versions.
```
Comment (bashofmann) [inline rancher-common/README.md:34]: "Used in the case of older Rancher versions." can be removed I think. The variable is always used.

**Review by bashofmann — APPROVED** [review]
Really strange that it worked without errors in my tests. But it probably just fell back to the default ingress controller certificate and I did not notice that.

**Comment by bashofmann** [comment]
Just tried this out on digitalocean and I get this error:

```
Error: Failed to unmarshal manifest: EOF

  on ../rancher-common/kubernetes.tf line 8, in resource "k8s_manifest" "cert_manager_crds":
   8: resource "k8s_manifest" "cert_manager_crds" {
```

**Comment by bashofmann** [comment]
The problem seems to be that the 0.12.0 crds have a trailing "---" at the end of the file.

File: rancher-common/README.md (hunk 1 above)
Comment (nikkelma) [reply rancher-common/README.md:34]: Agreed, was originally there to justify its existence as opposed to hard-coding all versions, seems reasonable to remove it.

**Review by bashofmann — APPROVED** [review]

**Pull request merged**